DataLayer/CmuData/CmuData.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <QList>
+
+#include "I_CmuUnit.h"
+
+class CmuData: I_CmuData
+{
+public:
+    CmuData(const unsigned char& numberOfCells);
+
+    I_CmuUnit getCmuUnit(const unsigned char& index);
+
+private: 
+    QList<I_CmuUnit*> cmuUnits_;
```

Comment:
@enochtsang Not sure if you're ok with me using pointers here (they seem to have been avoided throughout the project). Anyways let me know